python/ray/serve/_private/deployment_info.py

```diff
@@ -20,6 +21,9 @@ def __init__(
         version: Optional[str] = None,
         end_time_ms: Optional[int] = None,
         route_prefix: str = None,
+        # TODO(Ziy1-Tan): Passing `docs_path` to DeploymentInfo is deprecated
+        # and will be removed in next major version. It is updated when the
+        # FastAPI app is instantiated on the replica instead.
```

Comment:
`DeploymentInfo` is a private object, so we don't need to go through a deprecation process. As long as we can confirm there is no more internal usage of `DeploymentInfo.docs_path`, we should remove it entirely.